korman/properties/modifiers/render.py

```diff
@@ -86,3 +87,88 @@ def export(self, exporter, bo, so):
     @property
     def resolution(self):
         return int(self.quality)
+
+class PlasmaViewFaceMod(PlasmaModifierProperties):
+    pl_id = "viewfacemod"
+
+    bl_category = "Render"
+    bl_label = "Swivel"
+    bl_description = "Swivel object to face the camera, player, or another object"
+
+    preset_options = EnumProperty(name="Type",
+                           description="Type of Facing",
+                           items=[
+                                  ("Billboard", "Billboard", "Face the camera (Y Axis only)"),
+                                  ("Sprite", "Sprite", "Face the camera (All Axis)"),
+                                  ("Custom", "Custom", "Custom Swivel"),
+                            ])
+
+    follow_mode = EnumProperty(name="Target Type",
+                           description="Target of the swivel",
+                           items=[
+                                  ("kFaceCam", "Camera", "Face the camera"),
+                                  ("kFaceList", "Listener", "Face listeners"),
+                                  ("kFacePlay", "Player", "Face the local player"),
+                                  ("kFaceObj", "Object", "Face an object"),
+                            ])
+    target_object = StringProperty(name="Target Object",
+                                 description="Object to face")
+
+    pivot_on_y = BoolProperty(name="Pivot on local Y",
+                              description="Swivel only around the local Y axis",
+                              default=False)
+
+    offset = BoolProperty(name="Offset", description="Use offset vector", default=False)
+    offset_local = BoolProperty(name="Local", description="Use local coordinates", default=False)
+    offset_coord = FloatVectorProperty(name="", subtype="XYZ")
+
+    def created(self, obj):
+        self.display_name = obj.name
+
+    def export(self, exporter, bo, so):
+        vfm = exporter.mgr.add_object(pl=plViewFaceModifier, so=so, name=self.display_name)
```

Comment:
let's use `find_create_object` because other part of the plugin love to touch random modifiers.